modules/utils/src/rpc.ts

```diff
@@ -1,7 +1,7 @@
 import { ChannelRpcMethod, ChannelRpcMethodsPayloadMap, EngineParams } from "@connext/vector-types";
 
 export const constructRpcRequest = <T extends ChannelRpcMethod>(
-  method: T,
+  method: any,
```

Comment:
Should be typed as `T` right?

Reply:
I could swear I tried this last time and it didn't work that's why I had to change it but it seemed to work now.